packages/lit-dev-tools-cjs/src/playground-plugin/plugin.ts

```diff
@@ -263,6 +264,73 @@ export const playgroundPlugin = (
     return `<litdev-switchable-sample>${content}</litdev-switchable-sample>`;
   });
 
+  const neverReach = (_value: never, error: string): never => {
+    throw new Error(error);
+  };
+
+  /**
+   * An aside for extra information.
+   *
+   * Usage:
+   *
+   *   {% aside "positive" %}
+   *
+   *   Here is some content! This line is bolded.
+   *
+   *   This line is not bolded but on the same line as the previous one.
+   *
+   *   This line is another paragraph
+   *
+   *   {% endaside %}
+   *
+   *   {% aside "info" "no-header" %}
+   *
+   *   This one does not have a bolded header.
+   *
+   *   {% endaside %}
+   */
+  eleventyConfig.addPairedShortcode(
+    'aside',
+    (content: string, variant: AsideVariant, noHeader = '') => {
+      const acceptableVariants: ['info', 'warn', 'positive', 'negative'] = [
+        'info',
+        'warn',
+        'positive',
+        'negative',
+      ];
+
+      // If statement needs to be written this way to assert exhaustive check.
+      if (
+        acceptableVariants[0] !== variant &&
+        acceptableVariants[1] !== variant &&
+        acceptableVariants[2] !== variant &&
+        acceptableVariants[3] !== variant
+      ) {
+        // This will throw an error at runtime if it does not match and will
+        // throw a TS build time error if we add another variant and forget to
+        // update this file.
+        neverReach(
+          variant,
+          `Invalid {% aside ${variant} %}.` +
+            ` variant "${variant}" is not an acceptable variant of:` +
+            ` ${acceptableVariants.join(', ')}.`
+        );
+      }
```

Comment:
I wouldn't act on this. For fun I attempted this using a switch. Attempted some code gold.

```
      switch(variant) {
        case acceptableVariants[0]:
        case acceptableVariants[1]:
        case acceptableVariants[2]:
        case acceptableVariants[3]:
          break;
        default:
          // This will throw an error at runtime if it does not match and will
          // throw a TS build time error if we add another variant and forget to
          // update this file.
          neverReach(
            variant,
            `Invalid {% aside ${variant} %}.` +
              ` variant "${variant}" is not an acceptable variant of:` +
              ` ${acceptableVariants.join(', ')}.`
          );
      }
```

Reply:
yeah I usually do switches. I wonder how they minify in comparison.